### DISCUSSION TITLE
PDFSearchTool  ValueError : No data found

### DISCUSSION THREAD

**Discussion by gergirod** [crewAIInc/crewAI#1219] [Q&A]
this is what I'm doing in my code.
pdf_tool =PDFSearchTool(pdf="path to pdf")

Error

python3.12/site-packages/embedchain/loaders/pdf_file.py", line 21, in load_data
    raise ValueError("No data found")
ValueError: No data found

**Comment by zinyando** [comment]
> this is what I'm doing in my code. pdf_tool =PDFSearchTool(pdf="path to pdf")
> 
> Error
> 
> python3.12/site-packages/embedchain/loaders/pdf_file.py", line 21, in load_data raise ValueError("No data found") ValueError: No data found

Are you sure the path to the PDF is correct? Try putting the PDF in the same directory as your python script and try again.
